doubly_linked_list/doubly_linked_list.py

```diff
@@ -44,22 +44,55 @@ def __len__(self):
     return self.length
 
   def add_to_head(self, value):
-    pass
+    new_head = ListNode(value)
+    if (self.head == None):
+      self.tail = new_head
+      self.head = new_head
+    else:
+      new_head.next = self.head
+      self.head.prev = new_head
+      self.head = new_head
+    self.length += 1
+      
```

Comment:
Well done Itel; think about returning the old head too in another function, for reference sake. Maybe not; I think you did a great job here!